client/views/profile.html

```diff
@@ -81,63 +81,79 @@
                   </li>
                 </ul>
               </div>
-              <div class="tab-content container">
-                <div ng-show="tabIsActive(1)" class="col s12">
-                  <h5 class="text-center games-won">Games Won: <span>{{gamesWon}}</span></h5>
+              <div class="tab-content">
+                <div ng-show="tabIsActive(1)" class="col s12" ng-class="{'empty': gamesPlayed == 0}">
+                  <h5 class="text-center games-won" ng-show="gamesPlayed != 0">Games Won: <span>{{gamesWon}}</span></h5>
                   <br>
-                      <table class="striped">
-                          <thead>
-                            <tr>
-                              <th>Date</th>
-                              <th>Number of Players</th>
-                              <th>Winner</th>
-                            </tr>
-                          </thead>
-                  
-                          <tbody>
-                            <tr ng-repeat="game in gamesPlayed">
-                              <td>{{game.datePlayed}}</td>
-                              <td>{{game.players.length}}</td>
-                              <td>{{game.winner}}</td>
-                            </tr>
-                          </tbody>
-                        </table>
+                  <div class="empty-section no-games-played container" ng-show="gamesPlayed == 0">
+                    Welcome Newbie. <a href="/#!/app">Join a Game</a>
+                  </div>
+                  <div class="games-card-group" ng-show="gamesPlayed != 0">
```

Comment:
Quick question, is it possible for the variable gamesPlayed to be less that zero ?. If it is, then I think your checks should be "> 0" and not "!= 0".

Reply:
For some reason that doesn't work. Tried it already and I just tried it again. 
<img width="719" alt="screen shot 2018-07-12 at 10 57 27 pm" src="https://user-images.githubusercontent.com/22734663/42661742-fc9df696-8626-11e8-8461-d1cdb75fd1ac.png">
<img width="1241" alt="screen shot 2018-07-12 at 10 57 53 pm" src="https://user-images.githubusercontent.com/22734663/42661758-0ad3eee6-8627-11e8-94ec-4a84632c11e2.png">
<img width="822" alt="screen shot 2018-07-12 at 10 58 29 pm" src="https://user-images.githubusercontent.com/22734663/42661795-29dacdbe-8627-11e8-8fda-c894ce7cc4ab.png">
<img width="938" alt="screen shot 2018-07-12 at 10 58 45 pm" src="https://user-images.githubusercontent.com/22734663/42661800-30ad6174-8627-11e8-8c04-37aaee34eb66.png">